.github/workflows/testing.yml

```diff
@@ -17,32 +17,24 @@
          required: false
 
  jobs:
-   linting:
-      runs-on: ubuntu-latest
-      steps:
-        - uses: actions/checkout@v4
-        - uses: actions/setup-python@v4
-        - uses: pre-commit/action@v3.0.0
-
    testing:
-     needs: linting
      name: ${{ matrix.os }}, ${{ matrix.environment-file }}
      runs-on: ${{ matrix.os }}
      timeout-minutes: 30
      strategy:
        matrix:
          os: [ubuntu-latest]
          environment-file:
-           - ci/38-BASE.yaml
-           - ci/39-BASE.yaml
-           - ci/310-BASE.yaml
-           - ci/311-BARE.yaml
-           - ci/311-BASE.yaml
-           - ci/311-DEV.yaml
+           - ci/310-oldest.yaml
+           - ci/310-no-optional.yaml
+           - ci/310-stable.yaml
+           - ci/311-stable.yaml
+           - ci/312-stable.yaml
```

Comment:
We should standardise this. In libpysal we have no suffix on what is called `stable` here, momepy is using `latest`...